packages/gafl-webapp-service/src/pages/renewals/identify/identify.njk

```diff
@@ -12,6 +12,14 @@
 
 {%
     set errorMap = {
+        'referenceNumber' : {
+            'string.empty': { text: 'Enter the last six digits of your licence number', ref: '#ref' },
+            'string.pattern.base': { text: 'The last six digits of your licence number doesn\'t look right. Check and enter again', ref: '#ref' },
+            'string.invalid': {
+                text: 'The details you entered do not match those we have on record for licence number ' + payload.referenceNumber + '. Please check you have entered the correct date of birth and postcode',
```

Comment:
This message really needs changing - it implies something that isn't true

Reply:
Corrected grammar and spelling otherwise I can see nothing wrong with it.

Reply:
> The details you entered do not match those we have on record for licence number ' + payload.referenceNumber + '. Please check you have entered the correct date of birth and postcode

This implies that we know that the licence number exists and that the dob/postcode is incorrect which is not the case.

Reply:
Yes it was developed initially for the link on the email so the licence number will be found. In the case of the letter where the user types it does not do anything different.

Reply:
I would have to alter the error messaging to know which route it took. Right now I am not sure this is a high priority

Reply:
Surely both routes will not check the licence number existence in isolation of the other fields?  Otherwise this is a security issue.

Reply:
No its just the message - once structural validation has occurred the the items are checked as a tuple. The message was written for the email link which contains the licence number, so its was fair to assume that the licence number was correct and the message reflected that. It has since been changed to allow for the letter (generic) renewal so the error message was no longer appropriate. I have simplified the message now so it should apply to both